### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by dbinoj** [camsaul/django-rest-params#2]
Does this plugin support Python 3?
I'm getting this error: 
```
  File "/home/dbinoj/.virtualenvs/py3/lib/python3.5/site-packages/django_rest_params/decorators.py", line 20, in params
    VALID_TYPES = int, float, str, unicode, bool
NameError: name 'unicode' is not defined
```

**Comment by dbinoj** [comment]
Wrote a quickfix for this in PR #3

**Comment by camsaul** [comment]
Fixed by #3
